apps/api/src/app/events/usecases/add-job/add-job.usecase.ts

```diff
@@ -22,25 +22,39 @@ export class AddJob {
   ) {}
 
   public async execute(command: AddJobCommand): Promise<void> {
+    Logger.debug('command is: ' + command);
```

Comment:
I'm a bit concerned with PII here, and also, in Javascript, this will show: `command is: [Object object]` due to the way how strings works.

How are we dealing with PII leaking to those logs?

Reply:
Well to be fair, we do need to be carefull about that. 
The best approach I have seen is create dto variable annotations that will mask the type for us.

Reply:
@scopsy I think this should be addressed in an update soon that provides masking annotations to the dtos.

Reply:
This is now covered by the Log decorator.